config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -944,6 +944,63 @@ presubmits:
           operator: "Equal"
           value: "sig-node"
           effect: "NoSchedule"
+  - name: pull-kubernetes-node-kubelet-serial-podresourcesapi
+    cluster: k8s-infra-prow-build
+    always_run: false
+    optional: true
+    skip_report: false
+    skip_branches:
+    - release-\d+\.\d+  # per-release image
+    annotations:
+      testgrid-dashboards: sig-node-presubmits
+      testgrid-tab-name: pr-kubelet-serial-e2e-podresourcesapi
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    decorate: true
+    decoration_config:
+      timeout: 240m
+    path_alias: k8s.io/kubernetes
+    extra_refs:
+    - org: kubernetes
+      repo: release
+      base_ref: master
+      path_alias: k8s.io/release
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250714-70266d743a-master
+          resources:
+            limits:
+              cpu: 4
+              memory: 6Gi
+            requests:
+              cpu: 4
+              memory: 6Gi
+          command:
+            - runner.sh
+            - /workspace/scenarios/kubernetes_e2e.py
+          args:
+            - --deployment=node
+            - --gcp-zone=us-central1-b
+            - --node-args=--image-config-file=/home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/containerd/image-config-serial-resource-managers.yaml
```

Comment:
is this image the reason we need a separate lane and cannot run them as part of other Serial?

Reply:
we only need a filter, be it focus or label-filter which captures the podresourcesapi tests explicitly. Everything else is incidental and I can change as needed

Reply:
perhaps running as part of serial would be OK than? So we avoid an extra test lane

Reply:
That's surely a tradeoff we can explore. I'm fixing this PR as suggested by review comments, then we can evaluate another approach.
From CI costs perspective my intuition is that a targeted lane is less costly than running a full lenghtier lane only because we are interested in a subset of tests, but I'm totally open to discuss it.